<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>461</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>tab</string>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_title">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_url">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>url</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_url">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_send_tab">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_clear">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_delAllTabs">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>delAllTabs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_refresh">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>reFresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_settings">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>settings</string>
    </property>
   </widget>
  </widget>
  <action name="actionsettings">
   <property name="text">
    <string>settings</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
